### DISCUSSION TITLE
How to zoom out more to use a TUI

### DISCUSSION THREAD

**Discussion by Elevator8-source** [isontheline/pro.webssh.net#946] [Q&A] [answered]
I was using a TUI but now it says that the TUI is not usable anymore because the screen of my phone isn't big enough or i can't zoom out enough. How can I fix this? Please help

**Comment by isontheline** [comment]
What's a TUI?

You can pinch out / in to change the font size. Could you make a try?

**Reply by Elevator8-source** [reply]
A TUI is a Terminal User Interface. It is a very rudimentary way of having a GUI in a terminal. I zoomed out by pinching out as much as possible. I just miss 1 or 2 more zoom out steps to be able to few the TUI.

**Reply by isontheline** [reply]
Thank you I now understand 😁

OK you need to have a smaller font size than the currently allowed one, what's right?

**Reply by Elevator8-source** [reply]
Yes I think so

**Reply by Elevator8-source** [reply]
It did use to work, so maybe the TUI or WebSSH changed something

**Reply by isontheline** [reply]
I don't remind have changed something about the font size since weeks.
Currently font-size could be reduced down to 6px. Are you sure you will be able to read a so small font?

**Reply by Elevator8-source** [reply]
6px is no problem for me, I don’t know what to do to enable the TUI

**Reply by isontheline** [reply]
I'm sorry I don't understand 😞

Please share screenshots

**Comment by Elevator8-source** [comment]
Hi Arnaud, I am still having problems. Please check the image I attached. I would not like to switch to close sources Termius.

Kind regards,
Quintijn

**Reply by isontheline** [reply]
Hello @SgtDinosaur 👋

Please post your image over GitHub as email will drop it.

**Reply by Elevator8-source** [reply]
> Hello @SgtDinosaur 👋
> 
> 
> 
> Please post your image over GitHub as email will drop it.

![image](https://github.com/user-attachments/assets/47fa309d-8163-4aad-8644-fcd09b5383f8)

**Reply by isontheline** [reply]
I will check if I can reduce the minimum font size and will come back to you

**Reply by Elevator8-source** [reply]
Great thank you! Looking forward to your response.

**Reply by isontheline** [reply]
@SgtDinosaur Do you know the minimal size required by your TUI? (Cols x Rows)

**Reply by Elevator8-source** [reply]
@isontheline  I have received this photo from someone using a different ssh client on android. So I do not know if this is correct ![image](https://github.com/user-attachments/assets/43f0766c-d8ea-4c6d-9519-e11790e4d452)

**Comment by isontheline** [comment]
Please choose 6px font size on your connection settings :
<img width="407" alt="image" src="https://github.com/user-attachments/assets/9f87200d-e489-4076-aa33-0c37da9fd89e">

**Reply by Elevator8-source** [reply]
I already had it in 6px. Might it not be the px size but the limit on the zoom?
![image](https://github.com/user-attachments/assets/f7cc349f-1bf7-4c4a-9c91-4d56f8233728)

**Comment by isontheline** [comment]
Are you running on a « mini » device form factor?
Perhaps I can try to add more small font sizes

**Reply by Elevator8-source** [reply]
I am using an iPhone 15 but I have had tested this and had the same problem on a XR and an iPhone 12. A smaller font size could help. Or do you think it has to do with size requirement that I send you the screenshot of. Or are font sizes and zoom/screen size the same thing? How does this work technically?

**Reply by isontheline** [reply]
Technically the font size and zoom are interdependent, if you pinch in or out it will decrease / increase font size
Let's try with 4px font size, I will push a beta release tonight

**Reply by Elevator8-source** [reply]
Awesome, do I have to do something to receive it? Btw if this succeeds I will promote you to the staking community. They really need an open-source tool that can handle this TUI. There are no good open-source alternatives right now. Keep up the great work!

**Comment by Elevator8-source** [comment]
I have also tried to add this in the config file but it did not work.

![image](https://github.com/user-attachments/assets/c55d7aa4-e21b-4e04-bfc5-41295db5dcde)

I have deleted it now and will test it after you have pushed the update. Thanks man!

**Reply by isontheline** [reply, marked as answer]
> Awesome, do I have to do something to receive it? 

I will send you a link when available

> Btw if this succeeds I will promote you to the staking community. They really need an open-source tool that can handle this TUI. There are no good open-source alternatives right now. Keep up the great work!

Thank you 🙏

> I have also tried to add this in the config file but it did not work.

Very good catch! I was missing it!

Please try this instead :
```
Host *
  #!FontSize 4
```

Just tested on my side, it works

Keep the sharp # symbol (special command), add tab or spaces before the command, put the command under a hostname or catch all (eg examp’e)

**Reply by Elevator8-source** [reply]
It works! Even with font 5. Thank you for the tips. It does work kind of buggy. When I zoom in I cannot zoom back out to the same font/size as when I entered the terminal and the TUI is not able to be shown again.

**Reply by isontheline** [reply]
It's due to the fact I need to allow lower font size than 6px

**Reply by isontheline** [reply]
Please try this beta release : 
https://testflight.apple.com/join/QSrBK59z

**Reply by Elevator8-source** [reply]
It works like a charm! I bought the paid version. Thank you!

**Reply by isontheline** [reply]
Thank you so much @SgtDinosaur for your support about my work 🙏
